packages/manager/modules/exchange/src/account/accountTypes.service.js

```diff
@@ -38,10 +38,6 @@ export default class ExchangeAccountTypes {
 
     switch (formattedAccountType) {
       case this.TYPES.BASIC:
-        if (!this.CAN_DO.BASIC()) {
```

Comment:
can't you just update the condition here :
https://github.com/ovh/manager/blob/409d42926d36d13ef21924d308d5d78dc385ad4f/packages/manager/modules/exchange/src/account/accountTypes.service.js#L21
instead of removing the check ?

Reply:
Why not, but the question is: is this legit to make the test here ? As the method is here to get the value to display, regarding the type, why do we need to make this test to just display `Basic` text ?

Reply:
sure i was wondering if it's legit to remove the test but in case of displaying the value i think you are right it can be removed